learning_observer/learning_observer/google.py

```
@@ -114,14 +115,18 @@ def extract_parameters_from_format_string(format_string):
     return [f[1] for f in string.Formatter().parse(format_string) if f[1] is not None]
 
 
-async def raw_google_ajax(request, target_url, **kwargs):
+async def raw_google_ajax(runtime, target_url, **kwargs):
     '''
     Make an AJAX call to Google, managing auth + auth.
 
-    * request is the aiohttp request object.
+    * runtime is a Runtime class containing request information.
     * default_url is typically grabbed from ENDPOINTS
     * ... and we pass the named parameters
     '''
+    if isinstance(runtime, learning_observer.runtime.Runtime):
+        request = runtime.get_request()
+    else:
+        request = runtime
```

This should raise an exception, not set `request` to `runtime`. Alternatives:

- They could both be keyword parameters (default `None`). 
- We could always pass in a runtime.

A short-term alternative would be to print a warning, so we can clean up legacy code paths where this is still `request`. 

`request = runtime` is a bug.